.circleci/config.yml

```diff
@@ -0,0 +1,17 @@
+version: 2.1
+jobs:
+  check:
+    docker:
+      - image: circleci/node:12.16.3
+    steps:
+      - checkout
+      - run:
+          name: Install dependencies
+          command: yarn install
```

Comment:
I think we should use `-- frozen-lockfile` to ensure the behavior is the same as on local machines, see
https://github.com/Integreat/integreat-app/blob/main/.circleci/src/commands/restore_yarn_cache.yml
Not sure how many dependencies there already are whether a cache restoring makes sense and decreases the runtime.

Reply:
Good point. Added it.